modules/backend/formwidgets/taglist/partials/_taglist.htm

```diff
@@ -7,7 +7,8 @@
     id="<?= $field->getId() ?>"
     name="<?= $field->getName() ?>[]"
     class="form-control custom-select <?= !count($fieldOptions) ? 'select-no-dropdown' : '' ?> select-hide-selected"
-    <?php if ($customSeparators): ?>data-token-separators="<?= $customSeparators ?>"<?php endif ?>
+    data-token-separators="<?= $customSeparators ?? ''; ?>"
```

Comment:
Please leave this line the way that it was, doing it this way leaves an empty attribute on the generated HTML that I would prefer to avoid. @GinoPane

Reply:
@LukeTowers ok, no problem, done and applied the same for placeholder attribute